Tensile/Source/client/source/ReferenceValidator.cpp

```diff
@@ -456,18 +456,8 @@ namespace Tensile
             Type const* resultData      = resultBuffer + elementsBeforeData;
             Type const* resultAfterData = resultData + tensor.totalAllocatedElements();
 
-            int printed = 0;
```

Comment:
cppcheck: 
- Variable 'doPrint' is assigned a value that is never used.
- Variable 'errors' is assigned a value that is never used.
- Variable 'printedPreBuffer' is assigned a value that is never used.
- Variable 'printedInsideBuffer' is assigned a value that is never used.
- Variable 'printedPostBuffer' is assigned a value that is never used.